backend/Routes/Controllers/auth/middlewares/verifyUser.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const jwt = require('jsonwebtoken');
+
```

Comment:
This middleware should ideally add username property to body if token is correct, else it should send error. It shouldn't send a response if a correct token is passed,  rest of developers will use this middleware to modify their request body, and use the request body object in their code.